.github/workflows/ci.yml

```diff
@@ -30,7 +36,7 @@ jobs:
         run: yarn outdated
         continue-on-error: true
```

Comment:
It seems that this is not only configured for this one step, but for every other below.
Like a switch case fallthrough?! 🤔

Reply:
I have no experience with GitHub Actions yet so I don't know tbh. I'm thinking that we can remove `yarn outdated` from the ci flow and maybe check on dependabot or something like that if we can configure it to only create PR's for MAJOR and possibly MINOR version and not for PATCH versions as it seems to do by default which is just super annoying.